pkg/util/env/env.go

```diff
@@ -40,22 +43,28 @@ const (
 // GetNodeName returns the node's name used in Kubernetes, based on the priority:
 // - Environment variable NODE_NAME, which should be set by Downward API
 // - OS's hostname
-func GetNodeName() (string, error) {
-	nodeName := os.Getenv(NodeNameEnvKey)
+func GetNodeName() (nodeName string, err error) {
+	defer func() {
+		if runtime.GOOS == "windows" {
+			nodeName = strings.ToLower(nodeName)
+		}
+	}()
+	nodeName = os.Getenv(NodeNameEnvKey)
 	if nodeName != "" {
-		return nodeName, nil
+		return
 	}
 	klog.Infof("Environment variable %s not found, using hostname instead", NodeNameEnvKey)
-	var err error
+	nodeName = os.Getenv(ExternalNodeEnvKey)
+	if nodeName != "" {
+		return
+	}
+	klog.Infof("Environment variable %s not found, using hostname instead", ExternalNodeEnvKey)
```

Comment:
Better to print NodeNameEnvKey?

Reply:
NodeNameEnvKey is printed in line 56, maybe I should remove line 56 and change to NodeNameEnvKey in this line.